packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -186,6 +187,15 @@ export const HelpCenterContactForm = () => {
 	const [ debouncedMessage ] = useDebounce( message || '', 500 );
 	const [ debouncedSubject ] = useDebounce( subject || '', 500 );
 
+	const [ , experimentAssignment ] = useExperiment(
```

Comment:
We need to handle the loading state, otherwise users that have a loading assignment will get control experience, which will suddenly flick-over once the assignment loads.